build/ci/run_olm_verification.sh

```diff
@@ -16,5 +16,13 @@
 # limitations under the License.
 #
 
-docker build -f build/ci/Dockerfile.courier -t operator-courier .
-docker run --rm -t operator-courier
+docker build -f build/ci/Dockerfile.olm-verification -t operator-olm-verification .
+
+ARCH=$(uname -m)
+
+if [ "${ARCH}" != "ppc64le" ]; then
+  DOCKER_PATH=$(which docker)
```

Comment:
could the `which docker` command be used in the `if` instead of `ARCH`? it could be more accurate this way

Reply:
what do you mean? `which docker` command returns the docker path and not the arch

Reply:
yes, but shouldn't we care only about whether docker is installed or not?

Reply:
as it is currently defined, docker path can be found for for all three Jenkins agents
but we can not mount `/var/run/docker.sock` for Power

Reply:
got it